ext/standard/streamsfuncs.c

```diff
@@ -788,6 +789,11 @@ PHP_FUNCTION(stream_select)
 
 	PHP_SAFE_MAX_FD(max_fd, max_set_count);
 
+	if (secnull && !usecnull) {
+		zend_argument_value_error(4, "must not be null if argument #5 ($microseconds) is specified and non-null");
```

Comment:
```suggestion
		zend_argument_value_error(4, "cannot be null if argument #5 ($microseconds) is non-null");
```

or in order to get rid of the double negation:

```suggestion
		zend_argument_value_error(4, "must be null when argument #5 ($microseconds) is null");
```

I believe these formats follow the style we generally use

Reply:
Adjusted the message, but swapped the arguments, so it complains about argument 5 instead of argument 4.

Reply:
Yeah, that's a good idea!